src/commands/up.rs

```diff
@@ -98,6 +98,10 @@ pub struct UpCommand {
     /// All other args, to be passed through to the trigger
     #[clap(hide = true)]
     pub trigger_args: Vec<OsString>,
+
+    /// Only run a subset of the components
+    #[clap(long = "include_components", short = 'c')]
+    pub include_components: Vec<String>,
```

Comment:
I may be misreading this but it looks like this doesn't apply if the app is being loaded from a bindle, only if it's being loaded from a local file.  Could we add a `conflicts_with = BINDLE_ID_OPT` to the clap attribute please?

Reply:
Ah. I couldn’t find that attribute. No problem.